core/templates/dev/head/components/forms/schema_editors/SchemaBasedUnicodeEditorDirective.js

```diff
@@ -91,6 +92,10 @@ oppia.directive('schemaBasedUnicodeEditor', [
             if (!$scope.uiConfig()) {
               return '';
             } else {
+              if (!$scope.uiConfig().placeholder && (
+                  deviceInfoService.hasTouchEvents())) {
```

Comment:
Nit: extraneous parentheses around deviceInfoService.hasTouchEvents()

Reply:
Uhm.. how come? The parentheses are right.

Reply:
Does it still work when you do 
```
if (!$scope.uiConfig().placeholder && deviceInfoService.hasTouchEvents())
```
?

Reply:
@tjiang11 Yeah, but isn't there a 80 character per line limit?

Reply:
Ah yes, but I don't think JavaScript minds line breaks without parentheses right? I believe Python does that

Reply:
@tjiang11 Ohh. Yep, I just tried, apparently it doesn't. New things I learn about JS everyday :) . Changed it. PTAL!